src/pages/registration/contributors_tab/components/ContributorCard.tsx

```diff
@@ -47,7 +47,7 @@ const StyledContributorSection = styled.div`
 
 const StyledSequenceTextField = styled(TextField)`
   grid-area: sequence;
-  width: 3rem;
+  width: 3.6rem;
```

Comment:
fikk ikke til å resize TextField basert på hvor lang inputteksten er, så 3.6rem er bare det minste som klarer å vise 4 siffer

Reply:
Kunne dratt den ut i en const for å unngå magic number da